### PR TITLE
Fix navigation using links on legacy apps

### PR DESCRIPTION
**Pull request by lucasaarcoverde**
**What does this PR do?**
Fix the admin absolute navigation on legacy apps

**Steps to reproduce**
1. Go to this [workspace](https://lucasanv3--recorrenciaqa.myvtex.com/admin/Site/Produto.aspx)
2. Click the pricing or the logistic button (the green ones)
3. Check if the page loads

### CODE REVIEW

File: react/components/IframeLegacy.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const getLegacyBaseURL = (account, workspace) => {
   const isDevWorkspace = workspace && workspace !== 'master'
 
   const environment = isDevWorkspace ? `${workspace}--` : ''
-
+  console.log('workspace', workspace)
```
Comment (marcelovicentegc) [inline react/components/IframeLegacy.js]: 👀

File: react/components/IframeLegacy.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ class IframeLegacy extends Component {
   handleIframeMessage = (event) => {
     if (event.data && event.data.type) {
       const type = event.data.type
-
+      console.log('type', type)
```
Comment (marcelovicentegc) [inline react/components/IframeLegacy.js]: 👀

File: react/components/IframeLegacy.js (hunk 1 above)
Comment (lucasaarcoverde) [reply react/components/IframeLegacy.js]: :travolta:

**Review by marcelovicentegc — APPROVED** [review]
LGTM

**Review by kevinch — APPROVED** [review]

**Pull request merged**